### PR TITLE
feat-263: update details screens light mode

### PR DESCRIPTION
**Pull request by mgibson-casper**
### 🔥 Summary
https://github.com/casper-network/casper-blockexplorer-frontend/issues/263

### 📹 Video
**Account details**
Screenshot 2023-04-22 at 9 13 16 AM

**Deploy details**
Screenshot 2023-04-22 at 9 13 03 AM

**Block details**
Screenshot 2023-04-22 at 9 11 57 AM

### 😤 Problem / Goals
-

### 🤓 Solution
-

### 🗒️ Additional Notes
-

### CODE REVIEW

File: app/src/index.css [hunk 1]
```diff
@@ -1,11 +1,23 @@
-
 /* adds Inter & JetBrains Mono */
-@import url('https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800&family=JetBrains+Mono:wght@100;200;300;400;500;600;700&display=swap');
+/* @import url('https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800&family=JetBrains+Mono:wght@100;200;300;400;500;600;700&display=swap'); */
```
Comment (valfaro123) [inline app/src/index.css]: We can remove this old font import

**Review by valfaro123 — APPROVED** [review]

File: app/src/components/base/InfoCard/InfoCard.styled.ts [hunk 2]
```diff
@@ -18,16 +18,15 @@ export const InfoCardSection = styled.section`
...
   border-radius: 0.35rem;
   padding: 2rem;
   overflow-x: auto;
-  max-width: calc(100vw - 4rem);
-  margin: 0 auto 2rem auto;
+  margin-bottom: 2rem;
 
   @media only screen and (min-width: ${breakpoints.md}) {
-    max-width: calc(100vw - 6rem);
+    max-width: calc(100vw);
```
Comment (DorianKwan) [inline app/src/components/base/InfoCard/InfoCard.styled.ts]: We can remove this entirely if we don't have a max width anymore

Comment (DorianKwan) [reply app/src/components/base/InfoCard/InfoCard.styled.ts]: Or at the very least the calc isn't necessary here

File: app/src/components/buttons/NavbarItemLinkButton.tsx [hunk 3]
```diff
@@ -34,3 +35,8 @@ const DesktopNavItemLink = styled(Button)<{
   padding: 0;
   padding-bottom: 0.25rem;
 `;
+
+const NavItemWrapper = styled.div`
+  min-width: 90px;
```
Comment (DorianKwan) [inline app/src/components/buttons/NavbarItemLinkButton.tsx]: Can we use rem here?

File: app/src/components/cards/BlockDetailsCard/BlockDetailsCard.tsx [hunk 4]
```diff
@@ -31,10 +30,10 @@ export const BlockDetailsCard: React.FC<BlockDetailsCardProps> = ({
 
   const rawBlock = JSON.stringify(block);
 
+  // TODO: need BACK button to /blocks
```
Comment (DorianKwan) [inline app/src/components/cards/BlockDetailsCard/BlockDetailsCard.tsx]: Is this tracked in another ticket?

File: app/src/components/cards/BlockDetailsCard/BlockDetailsCard.tsx [hunk 5]
```diff
@@ -55,179 +54,203 @@ export const BlockDetailsCard: React.FC<BlockDetailsCardProps> = ({
...
 `;
 
-const HashHeading = styled(GradientHeading)<{ isTruncated: boolean }>`
-  font-weight: ${fontWeight.extraBold};
+const HashHeading = styled(Heading)<{ isTruncated: boolean }>`
+  font-weight: ${fontWeight.medium};
   display: inline;
   margin: 0;
-  width: ${({ isTruncated }) => (isTruncated ? '10%' : '100%')};
+  width: ${({ isTruncated }) => (isTruncated ? '40%' : '75vw')};
```
Comment (DorianKwan) [inline app/src/components/cards/BlockDetailsCard/BlockDetailsCard.tsx:234]: Check out the `ch` css unit.  It's quite good at being consistent at font or character changes

File: app/src/components/cards/DeployDetailsCard/DeployDetailsCard.tsx [hunk 6]
```diff
@@ -28,14 +27,14 @@ export const DeployDetailsCard: React.FC<DeployDetailsCardProps> = ({
...
   const { isMobile } = useAppWidth();
   const { t } = useTranslation();
 
   return (
-    <InfoCard>
-      <HeadContentWrapper>
-        <DeployHeading type="h1">{t('deploy-details')}</DeployHeading>
+    <>
+      <HeaderContent>
+        {/* TODO: should add a "back to" here (and in all other details pages) */}
```
Comment (DorianKwan) [inline app/src/components/cards/DeployDetailsCard/DeployDetailsCard.tsx]: Same question as before

File: app/src/components/styled/AvatarIcon/AvatarIcon.styled.ts [hunk 7]
```diff
@@ -1,8 +1,10 @@
...
 
 export const AvatarIcon = styled.img<{ isTruncated: boolean }>`
-  width: 3rem;
-  display: inline;
-  margin-right: 1.25rem;
-  margin-bottom: ${({ isTruncated }) => (isTruncated ? '1rem' : '5.166rem')};
+  width: ${pxToRem(74)};
+  margin: 0 2rem;
+  /* margin-right: 1.25rem; */
+  /* margin-bottom: ${({ isTruncated }) =>
```
Comment (DorianKwan) [inline app/src/components/styled/AvatarIcon/AvatarIcon.styled.ts]: Can we delete this?

File: app/src/index.css (hunk 1 above)
Comment (DorianKwan) [reply app/src/index.css]: +1

**Review by DorianKwan — CHANGES_REQUESTED** [review]

File: app/src/index.css (hunk 1 above)
Comment (mgibson-casper) [reply app/src/index.css]: Oops, yeah sorry guys I see I have lots of commented code. Meant to delete all that! Everything is cleaned up.

File: app/src/components/base/InfoCard/InfoCard.styled.ts (hunk 2 above)
Comment (mgibson-casper) [reply app/src/components/base/InfoCard/InfoCard.styled.ts]: We still want the vw here so just removed the calc.

File: app/src/components/buttons/NavbarItemLinkButton.tsx (hunk 3 above)
Comment (mgibson-casper) [reply app/src/components/buttons/NavbarItemLinkButton.tsx]: Yup yup!

File: app/src/components/cards/BlockDetailsCard/BlockDetailsCard.tsx (hunk 4 above)
Comment (mgibson-casper) [reply app/src/components/cards/BlockDetailsCard/BlockDetailsCard.tsx]: No but I'll add one. We decided not to go with this more now until we get a deploys page, and then we'll do both at once.

File: app/src/components/cards/BlockDetailsCard/BlockDetailsCard.tsx (hunk 5 above)
Comment (mgibson-casper) [reply app/src/components/cards/BlockDetailsCard/BlockDetailsCard.tsx:234]: Oh yeah I always forget about that guy. It actually works quite well with vw so I think I'll just go with that 👍

File: app/src/components/styled/AvatarIcon/AvatarIcon.styled.ts (hunk 7 above)
Comment (mgibson-casper) [reply app/src/components/styled/AvatarIcon/AvatarIcon.styled.ts]: Deleted.

File: app/src/index.css (hunk 1 above)
Comment (DorianKwan) [reply app/src/index.css]: Awesome!

File: app/src/components/buttons/NavbarItemLinkButton.tsx (hunk 3 above)
Comment (DorianKwan) [reply app/src/components/buttons/NavbarItemLinkButton.tsx]: Beautiful!

File: app/src/components/cards/BlockDetailsCard/BlockDetailsCard.tsx (hunk 5 above)
Comment (DorianKwan) [reply app/src/components/cards/BlockDetailsCard/BlockDetailsCard.tsx:234]: Okay sounds good

File: app/src/components/styled/AvatarIcon/AvatarIcon.styled.ts (hunk 7 above)
Comment (DorianKwan) [reply app/src/components/styled/AvatarIcon/AvatarIcon.styled.ts]: Awesome

**Review by DorianKwan — APPROVED** [review]

**Pull request merged**